src/Fable.ReactGoogleMaps/Fable.Helpers.ReactGoogleMaps.fs

```diff
@@ -58,6 +68,9 @@ type MapRef(mapRef) =
 
     member __.GetCenter() : Coordinates.Position =
         mapRef?getCenter() |> unbox
+        
+    member __.FitBounds(bounds: Coordinates.LatLngBounds) : unit =
```

Comment:
Can you please add an overload that takes the bounds that I already added in coordinates module?